app/views/tracks/show/joined.html.haml

```diff
@@ -0,0 +1,60 @@
+#page-track-dashboard
+  %nav.track-nav
+    .lg-container.container
+      = link_to "<-", tracks_path, class: "back"
+      = image_tag(@track.icon_url, class: 'c-track-icon')
+      .title= @track.title
+      .tabs
+        = link_to "#", class: 'selected' do
+          Overview
+
+        = link_to track_concepts_path(@track) do
+          Concepts
+
+        = link_to "#" do
+          Exercises
+
+  %header.track-header
+    .lg-container.container
+      = image_tag(@track.icon_url, class: 'c-track-icon')
```

Comment:
Probably want to add the `alt`

Reply:
I'm going to put all of these images into helpers and image_tag will be "broken" within Rails so it can't be used, so this will get done then. Thank you for the prompt though!